uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/leave-request-data.js

```diff
@@ -16,9 +16,9 @@ define([
         'type_id': absenceTypeData.all().values[0]['id'],
         'contact_id': '202',
         'status_id': optionGroupMock.specificValue('hrleaveandabsences_leave_request_status', 'value', '1'),
-        'from_date': '2016-02-02',
```

Comment:
this file should be `.data.js`, please refer to the style guide (and fix any other data file if needed)

Reply:
Fixed